### DISCUSSION TITLE
I can not install @tamagui/spinner with bun

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [tamagui/tamagui#3078] [Q&A]
Hi I am trying to install `@tamagui/spinner` with bun I get this error: 
`error: GET https://registry.npmjs.org/@tamagui%2fspinner - 404`

**Comment by GerardPolloRebozado** [comment]
Same with npm Not Found - GET https://registry.npmjs.org/@tamagui%2fspinner - Not found

**Comment by namdh03** [comment]
Same
